src/WinRT.Runtime/ExceptionHelpers.cs

```diff
@@ -154,6 +155,13 @@ private static Exception GetExceptionForHR(int hr, bool useGlobalErrorState, out
                     case E_ELEMENTNOTENABLED:
                         ex = new Microsoft.UI.Xaml.Automation.ElementNotEnabledException();
                         break;
+                    case ERROR_INVALID_WINDOW_HANDLE:
+                        ex = new System.Runtime.InteropServices.COMException(
+@"Invalid window handle. (0x80070578)
+Consider IWindowNative, IInitializeWithWindow
+See also https://aka.ms/cswinrt/interop#windows-sdk", 
```

Comment:
consider linking to Microsoft Docs instead? https://docs.microsoft.com/windows/apps/desktop/modernize/winrt-com-interop-csharp

Reply:
added mdocs link to interop md